wp-content/themes/cambodia-ict-camp/inc/shortcode/display-custom-post-type.php

```diff
@@ -3,125 +3,142 @@
 // [display_cpt post_type="post" show_thumbnail= true show_meta=true thumbnail_size="250, 200" show_post_title=true title_headtag ="h6" open_new_tab=false show_content=false show_excerpt=true display="inline/list" flex_box_row=true row_container=true custom_link=true no_link_get_permalink=true camp_year="" max_post="5" open_new_tab=]
 add_shortcode( 'display_cpt', 'display_custom_post_type');
 
-function display_custom_post_type($atts){
+/**
+ * Shortcode for displaying list of custom posts in Gridview or Listview
+ * 
+ * @param  $atts [List of short code attribute]
+ * @return [string]
+ */
+function display_custom_post_type( $atts ) {
     ob_start();
+
     $shortcode_atts = shortcode_atts( array(
-        'post_type' => 'post',
-        'show_thumbnail' => true,
-        'thumbnail_size' => '250, 200',
-        'show_post_title' => true,
-        'show_meta' => true,
-        'title_headtag' => 'h6',
-        'open_new_tab' => true,
-        'show_excerpt' => true,
-        'show_content' => false,
-        'display' => 'list', //inline/list
-        'row_container' => true, //if display=inline
-        'flex_box_row' => true, //if display=inline
-        'custom_link' => true,
+        'post_type'             => 'post',
+        'show_thumbnail'        => true,
+        'thumbnail_size'        => 'full',
+        'show_post_title'       => false,
+        'show_meta'             => true,
+        'title_headtag'         => 'h6',
+        'open_new_tab'          => true,
+        'show_excerpt'          => false,
+        'show_content'          => true,
+        'display'               => 'list', // inline/list
+        'row_container'         => true,   // if display=inline
+        'flex_box_row'          => true,   // if display=inline
+        'custom_link'           => true,
         'no_link_get_permalink' => true,
```

Comment:
Please tell me the reason of updating these default value.

Reply:
For `thumbnail_size` the value should be `medium`, `large`, `full`, or `array( width, hight)`. The other value is just my testing, and I forget to change it back. I just realize that the value of the shortcode can be set from Text Editor.

Reply:
Of course, the define size of thumbnail_size could be string and array. However, you can't define full size as the default. The reason, that width and hight are defined because the shortcode will be called and used the content editor, so the size image need to be more flexible. In fact, thumbnail value can define as the default value, however, the image is always cut off. Well, you welcome to improve the function of shortcode to support both string or array value. However, calling like this won't support array value. `get_the_post_thumbnail(` get_the_ID(), $shortcode_atts['thumbnail_size'] );` Please define the size the mostly use as default, but not full size. 
Note: Please test carefully, because updating shortcode attribute will affect to the using shortcode in existing content.

Reply:
phasing: '250, 200' as the value of the `thumbnail_size` does not determine the size of the logo.